cpp/src/arrow/ipc/reader.cc

```diff
@@ -715,7 +716,7 @@ Status ReadTensor(int64_t offset, io::RandomAccessFile* file,
   RETURN_NOT_OK(file->Seek(offset));
 
   std::unique_ptr<Message> message;
-  RETURN_NOT_OK(ReadContiguousPayload(file, &message));
+  RETURN_NOT_OK(ReadContiguousPayload(file, &message, true));
```

Comment:
When passing bare bool literals like that as argument value, is it possible to add a comment with the argument name?
Something like:
```cpp
  RETURN_NOT_OK(ReadContiguousPayload(file, &message, true /* aligned */));
```
(not sure the above passes linting, to be honest)

Reply:
Good point, I fixed it. If this doesn't pass, then I can just do 

```cpp
bool aligned = true;
RETURN_NOT_OK(ReadContiguousPayload(file, &message, aligned));
```